docs/apppromotepipeline.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Pipeline
+metadata:
+  name: apppromote-pipeline
```

Comment:
Again, maybe we want to use `app-promote` or `service-promote` which might be more readable?

Reply:
I change it.